acceptance/acceptance_test.go

```diff
@@ -95,13 +129,30 @@ func TestAccept(t *testing.T) {
 	testDirs := getTests(t)
 	require.NotEmpty(t, testDirs)
 
+	if singleTest != "" {
+		testDirs = slices.DeleteFunc(testDirs, func(n string) bool {
+			return n != singleTest
+		})
+		require.NotEmpty(t, testDirs, "singleTest=%#v did not match any tests\n%#v", singleTest, testDirs)
+	}
+
 	for _, dir := range testDirs {
 		testName := strings.ReplaceAll(dir, "\\", "/")
 		t.Run(testName, func(t *testing.T) {
-			t.Parallel()
+			if !InprocessMode {
+				t.Parallel()
+			}
+
+			if singleTest != "" && singleTest != testName {
+				t.Skip("Skipping due to singleTest")
+				return
+			}
```

Comment:
Not needed; filtering is done by the block above.

Reply:
Might not work on Windows because "singleTest" doesn't get the string replacement and "testName" does.

Reply:
Removed filter.

> Might not work on Windows because "singleTest" doesn't get the string replacement and "testName" does.

Should be fine if you use forward slashes in singleTest.